pkg/auth/tokens/manager.go

```diff
@@ -377,13 +376,34 @@ func (m *Manager) logout(actionName string, action *types.Action, request *types
 	}
 	w.Header().Add("Content-type", "application/json")
 
-	//getToken
-	status, err := m.deleteToken(tokenAuthValue)
+	storedToken, status, err := m.getToken(tokenAuthValue)
 	if err != nil {
-		logrus.Errorf("DeleteToken failed with error: %v", err)
-		if status == 0 {
+		logrus.Errorf("getToken failed with error: %v", err)
+		if status == http.StatusNotFound {
+			// 0
 			status = http.StatusInternalServerError
+			return httperror.NewAPIErrorLong(status, util.GetHTTPErrorCode(status), fmt.Sprintf("%v", err))
```

Comment:
Instead of `fmt.Sprintf`, you could just call `err.Error()` ?

Reply:
done. commit 3b5168e2824999c0f569ba78924028cf7a4af3d9